docs/pages/admin-guides/access-controls/sso/azuread.mdx

```diff
@@ -92,14 +92,18 @@ Before you get started, you’ll need:
 
    ![Put in Security group claim](../../../../img/azuread/azuread-8b-groupclaim.png)
 
-1. Add a claim that transforms the format of the Azure AD username to lower case, in order to pass it to
-   Teleport. Set the Source to "Transformation". In the new panel:
+1. (optional) Add a claim that transforms the format of the Azure AD username to lower case, in order to use it inside
```

Comment:
Should this say "Microsoft Entra ID"?

Reply:
Yes, as the entire guide should I suppose.

Reply:
I opened https://github.com/gravitational/teleport/issues/47776 to handle this ask as there's a lot of references that will need updating. I will try and get to it shortly.